api/scpca_portal/models/dataset.py

```diff
@@ -60,6 +61,22 @@ class Meta:
     data_hash = models.CharField(max_length=32, null=True)
     metadata_hash = models.CharField(max_length=32, null=True)
     readme_hash = models.CharField(max_length=32, null=True)
+    combined_hash = models.CharField(max_length=32, null=True)
```

Comment:
Let's rework combined hash to be a function that returns a tuple:

```python
data_hash, metadata_hash, readme_hash, combined_hash = get_hashes()
```

this should reduce the duplication of calculating the separate hashes as well and simplify usage